tools/build_file_generator/BuildFileGenerator.java

```diff
@@ -371,6 +371,9 @@ private CheckstyleTestExtension getCheckstyleTest(String target) {
       if (target.contains("code_review_service_grpc")) {
         target = target.replace("_service_grpc", "_java_grpc");
       }
+      if (target.contains("auth_service_grpc")) {
```

Comment:
Why can't we also use:
`target = target.replace("_service_grpc", "_java_grpc");`
Instead of:
`target = target.replace("_grpc", "_java_grpc");`
?

Reply:
The problem is in names. `auth_service.proto` already involve `service` in the name. Name of service is AuthService. (`proto file name = AuthService`). 
The second case has `code_review.proto`. Name of service: CodeReviewService (`proto file name != CodeReviewService`)

Reply:
Right. But when we parse proto files, we know all the data:
- proto filename
- java package
- service name

So maybe we can use that information to do this in a general way?

Reply:
Added the code to do this in a general way.